data/starts.txt

```diff
@@ -0,0 +1,215 @@
+	`	This "Aphid" could suit your needs. It is Fast, and able to carry a respectable amount of cargo.`
+	scene "thumbnail/hai aphid"
+	`	That is, except if you want to get adventurous and try out the impetious Grasshopper. Surely the interceptor would be able to beat human small warships prodided a competent pilot is at the helm...`
+	`	You have heard tales of pirates and quick money... Indeed, up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them strength...`
+	choice
+		`	(Go to the bank.)`
+	scene "scene/hai bank"
+	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
+	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some part of the ship in order to be able to afford flying it!`
+	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`
+	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, on your way to buy the Aphid, but you see a notice. A retiring captain offers to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
+	choice
+		`	(Go to the shipyard.)`
+			decline
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain that offers to sell the Grasshopper. Thankfully they did not yet sell it yet.`
+	`	The captain greets you in Hai. You somehow never really learned the language, probably due to the fact everyone speaks yours to some degree. Seeing you do not understand they switch to your language.`
+	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
+	choice
+		`	"Yes indeed."`
+		`	"How did you know?"`
+		`	"Sorry, not interested."`
+			decline
+	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" They say, taking you to the ship. It is not brand new, but it still looks in surprisingly good condition.`
+	scene "thumbnail/hai grasshopper"
+	`	"Because I like you I'll give you a friendly price. What do you say about... 1 millions, for this wonderful ship? That's less than a third price you would have to pay if you were to buy this model in a shipyard, except you can't find a model like this in shipyards, trust me!"`
```

Comment:
```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... one million flat, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```